ipa-macros/src/step/mod.rs

```diff
@@ -0,0 +1,32 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::parse_macro_input;
+
+/// Generate a state transition graph and the corresponding `StepNarrow` implementations for the
```

Comment:
are we still generating `StepNarrow` here? can't find it in this file

Reply:
I jumped ahead of myself. I'll fix the comment.